From 7f1b27e9521eee27435cd199eb94da1a99b0c4e4 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Mon, 14 Dec 2020 11:09:04 +0530
Subject: [PATCH] Enable AIDL interface for Light HAL

Changed user and group from nobody to system

Tracked-On: None
Signed-off-by: svenate <salini.venate@intel.com>
---
 light/aidl/default/Android.bp        | 2 +-
 light/aidl/default/lights-default.rc | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git light/aidl/default/Android.bp light/aidl/default/Android.bp
index ae3f4630d..d97a0b169 100644
--- light/aidl/default/Android.bp
+++ light/aidl/default/Android.bp
@@ -1,5 +1,5 @@
 cc_binary {
-    name: "android.hardware.lights-service.example",
+    name: "android.hardware.lights-service.intel",
     relative_install_path: "hw",
     init_rc: ["lights-default.rc"],
     vintf_fragments: ["lights-default.xml"],
diff --git light/aidl/default/lights-default.rc light/aidl/default/lights-default.rc
index 687ec97dd..30c2d5d95 100644
--- light/aidl/default/lights-default.rc
+++ light/aidl/default/lights-default.rc
@@ -1,5 +1,5 @@
-service vendor.light-default /vendor/bin/hw/android.hardware.lights-service.example
+service vendor.light-default /vendor/bin/hw/android.hardware.lights-service.intel
     class hal
-    user nobody
-    group nobody
+    user system
+    group system
     shutdown critical
-- 
2.17.1

